channel/handlers.go

```diff
@@ -2577,15 +2586,42 @@ func (server *Server) handlePartyEvent(conn mnet.Server, reader mpacket.Reader)
 			party.updateJobLevel(playerID, job, level)
 		}
 	default:
-		log.Println("Unkown party event type:", op)
+		log.Println("Unknown party event type:", op)
+	}
+}
+
+func (server *Server) handleChangeRate(conn mnet.Server, reader mpacket.Reader) {
```

Comment:
This function should also print the new rates to screen

Reply:
what do you mean 'to screen' ? as a log or to the players? 
cuz a msg to players is sent in another handler

Reply:
I meant a `println` statement so that it's displayed in the terminal.